docs/options.md

```diff
@@ -99,6 +99,16 @@ FAIL - Combined - Deployment hello-kubernetes has selector hello-kubernetes that
 
 This is just the tip of the iceberg. Now you can ensure that duplicate values match across the entirety of your configuration files.
 
+## `--config-file`
+
+Sometimes the working directory isn't central enough for conftest's config file.
```

Comment:
Please rephrase this to only state what it does. Something along the lines of "Specify the path to Conftest's configuration file."